.env.example

```
@@ -11,4 +11,6 @@ STRIPE_PRICE_ID=price_mock
 STRIPE_WEBHOOK_SECRET=whsec_mock
 APP_URL=backend-app-url
 POSTHOG_API_KEY=phc_your_project_api_key_here
-POSTHOG_HOST=https://us.i.posthog.com or your host
\ No newline at end of file
+POSTHOG_HOST=https://us.i.posthog.com or your host
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTHOG_HOST key should go before the STRIPE_PRICE_ID key

(UnorderedKey)